packages/js/src/Novu.ts

```diff
@@ -0,0 +1,39 @@
+import { NovuEventEmitter } from './event-emitter';
+import type { EventHandler, EventNames, Events } from './event-emitter';
+import { Feeds } from './feeds';
+import { Session } from './session';
+import { Preferences } from './preferences';
+
+interface NovuOptions {
+  applicationIdentifier: string;
+  subscriberId: string;
+  subscriberHash?: string;
+}
+
+export class Novu implements Pick<NovuEventEmitter, 'on' | 'off'> {
+  #emitter: NovuEventEmitter;
+  #session: Session;
+
+  public readonly feeds: Feeds;
+  public readonly preferences: Preferences;
+
+  constructor(options: NovuOptions) {
+    this.#emitter = new NovuEventEmitter();
+    this.#session = new Session(this.#emitter, {
```

Comment:
```suggestion
    this.#session = new Session(this.#emitter, options)
```

Reply:
there will be more properties and session doesn't need to know about them